src/components/LoggedInLayout/LoggedInLayout.jsx

```diff
@@ -14,26 +14,32 @@ class LoggedInLayout extends React.Component {
 
   componentDidMount() {
     this.updateWindowWidth();
-    window.addEventListener('resize', this.updateWindowWidth)
+    window.addEventListener("resize", this.updateWindowWidth);
   }
 
   updateWindowWidth = () => {
     this.setState({ windowWidth: window.innerWidth });
-  }
+  };
```

Comment:
These semicolons are also newly added. These are fine, but if you're going to add it here, can you make sure to add it in all places then so that we're consistent?